source/fundamentals/data-formats/serialization.txt

```diff
@@ -0,0 +1,166 @@
+.. _csharp-serialization:
+
+=============
+Serialization
```

Comment:
High-level suggestion after reading the whole page: Add a series of steps at the beginning of the page to describe the entire process (or something like that). The connection between the various moving pieces isn't always super clear.

Reply:
I restructured the page to have the serialization section first, that and removing the provider section makes it a little more clear I think. Let me know if you agree or think we still need to add some more clarification